onnxruntime/python/onnxruntime_validation.py

```diff
@@ -22,7 +22,7 @@ def check_distro_info():
         __my_distro__ = __my_system__
         __my_distro_ver__ = platform.release().lower()
 
-        if __my_distro_ver__ != "10":
+        if __my_distro_var__ not in ["10", "11"]:
```

Comment:
The var name is apparently wrong.

Reply:
Same system. But I will encounter this warning.

![image](https://github.com/microsoft/onnxruntime/assets/70036307/600dfb7d-77a2-41f7-848c-83a11680193c)

in Setting:
Version	Windows 11 pro
Version	23H2

![image](https://github.com/microsoft/onnxruntime/assets/70036307/c746bb69-fe2d-4ac7-99d0-f498ed86d314)

in system info:
Microsoft Windows 11 pro
10.0.22631 build 22631

![image](https://github.com/microsoft/onnxruntime/assets/70036307/08da5716-6529-41d3-9bd1-0b79c4cce288)

Reply:
Probably because you were using python 3.12.